### PR TITLE
ADman Media new bidder adapter

### PR DESCRIPTION
**Pull request by admanmedia**
- [x] New bidder adapter  

- test parameters for validating bids
```
{
  bidder: 'adman',
  params: {
    id: 5ae1b797
  }
}
```
- contact email: prebid@admanmedia.com
- [x] official adapter submission

### CODE REVIEW

**Comment by bretg** [comment]
Docs PR at https://github.com/prebid/prebid.github.io/pull/1209

File: modules/admanBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      bids,
+      deviceWidth: screen.width
+    };
+
+    if (bidderRequest && bidderRequest.gdprConsent) {
+      payload.gdpr = {};
+      payload.gdpr.consent = bidderRequest.gdprConsent.consentString;
+      payload.gdpr.applies = bidderRequest.gdprConsent.gdprApplies;
+    } else {
+      payload.gdpr.consent = '';
```
Comment (idettman) [inline modules/admanBidAdapter.js]: The assignment of `payload.gdpr.consent` to empty string is being overwritten by `null` on the next line. Either delete **line 35** (if this is working correctly server-side), or change it to a object literal like this:
```
payload.gdpr = {
  consent: ''
}
```

File: modules/admanBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import {registerBidder} from '../src/adapters/bidderFactory';
+import * as utils from '../src/utils';
+
+const BIDDER_CODE = 'adman';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: ['video', 'banner'],
+  isBidRequestValid: function(bid) {
+    let isValid = false;
```
Comment (idettman) [inline modules/admanBidAdapter.js]: The `isValid` logic can be simplified if you use `utils.deepAccess` to something like this:
```
isBidRequestValid: function(bid) {
  const isValid = _validateId(utils.deepAccess(bid, 'params.id'));
  if (!isValid) {
    utils.logError('Adman id parameter is required. Bid aborted.');
  }
  return isValid;
}
```

**Comment by idettman** [comment]
All changes look good, however I see there is a `console.log` used in the test file, this needs to be removed. Once this is completed, I'll approve

File: test/spec/modules/admanBidAdapter_spec.js [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+        gdprConsent: {
+          consentString: 'BOZcQl_ObPFjWAeABAESCD-AAAAjx7_______9______9uz_Ov_v_f__33e8__9v_l_7_-___u_-33d4-_1vf99yfm1-7ftr3tp_87ues2_Xur__59__3z3_NohBgA',
+          gdprApplies: true
+        }
+      });
+
+      expect(request.url).to.equal(ENDPOINT);
+      expect(request.method).to.equal('POST');
+
+      const payload = JSON.parse(request.data);
```
[the remark this answers is not in the document]
Comment (mkatogi) [reply test/spec/modules/admanBidAdapter_spec.js]: Removed. Thanks!

**Pull request merged**